README.md

```diff
@@ -4,6 +4,25 @@ This is an open API specification for issuing digital credentials based on stand
 
 The latest version can be viewed at https://w3c-ccg.github.io/vc-issuer-http-api/index.html.
 
+## Versioning
+
+This repo and specification use [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200).
+
+The API version is defined [api.yml](./api.yml).
+
+You can browse the [latest release of github](https://github.com/OR13/vc-issuer-http-api/releases).
```

Comment:
Does this refer to your local fork and not the main repo?

Reply:
my bad thats wrong.

Reply:
Fixed